mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -314,6 +316,7 @@ def _predict(model_uri, input_path, output_path, content_type):
     if output_path is None:
         predictions_to_json(pyfunc_model.predict(df), sys.stdout)
     else:
+        # Should we check if output_path is valid here? Define `valid`.
```

Comment:
```suggestion
```
If the output path is invalid, it will throw on writing the predictions, right? I think that's fine.

Reply:
That's true :)